oap-server-bom/pom.xml

```diff
@@ -73,7 +73,7 @@
         <awaitility.version>3.0.0</awaitility.version>
         <httpcore.version>4.4.13</httpcore.version>
         <commons-compress.version>1.21</commons-compress.version>
-        <banyandb-java-client.version>0.1.0</banyandb-java-client.version>
+        <banyandb-java-client.version>0.2.0-SNAPSHOT</banyandb-java-client.version>
```

Comment:
You need a repository to get this version, I think?

Reply:
Yes. Pending util we publish java client `0.2.0`.